### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewExplorer_ScreenReader: Screenreader does not read the necessary information to navigate within the treeview section.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52009]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.3.1
ScreenReader Versions : NVDA2018.1.1 

**Repro Steps:**
1. Launch VS Code.
2. Navigate to Activity Bar and select "Explorer"(Cntrl+Shift+E) button.
3. Navigate to "Open Editors" and "File Explorer" section treeview items using "Tab".
4. Read all the treeview items.

**Actual:**
NVDA does not read out the necessary instructions - to navigate within the treeview and move out of it. 

**Expected:**
NVDA should read out each instruction clearly related to navigation inside and moving outside the treeview.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The screenreader users will not be able to know the necessary instructions to navigate within the treeview and move out of it.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**

![a11y_vscode_viewexplorer_screenreader_treeviewnavigation](https://user-images.githubusercontent.com/39204909/41462356-35d05a26-70b0-11e8-9180-51deba348259.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of 2 other issues already created
